scripts/get-distro.sh

```diff
@@ -10,8 +10,8 @@ UNAME="$(uname -s)"
 case "$UNAME" in
     Darwin)
         DIST='macos'
-        VERSION_ID=$(sw_vers | gsed -n '/^ProductVersion:/p' | gsed -r 's/ProductVersion:(.*)/\1/g' | gsed -r 's/([0-9]+).*/\1/g' | gsed 's/^[ \t]*//g')
-        SYSTEM="$(echo "${DIST}${VERSION_ID}" | gsed -r 's/([a-zA-Z]*)-.*/\1/g')"
+        VERSION_ID="$(sw_vers | grep 'ProductVersion' | cut -d ':' -f2 | cut -d'.' -f1 | tr -d '\t')"
```

Comment:
To recover the previous behavior, we should `tr -d ' '` as well, I think

Reply:
fixed